bin/format_secrets.rb

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env ruby
+require 'json'
+
+data = JSON.parse(ENV['SSM_PARAMETERS_JSON'])
```

Comment:
What do you think is better here, using an env var or just receiving the json as a parameter in the script?

Reply:
Could be work as well, I can include the change to receive the json as parameter